### PR TITLE
OSD-10311 - Adding jobs for lint and code coverage for osd-network-verifier

### PR DESCRIPTION
**Pull request by bdematte**
Updating prow jost to add lint (presubmit) and code-coverage (pre and post submit)

### CODE REVIEW

File: ci-operator/jobs/openshift/osd-network-verifier/openshift-osd-network-verifier-main-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --secret-dir=/secrets/certman-operator-codecov-token
+        - --target=publish-coverage
+        command:
+        - ci-operator
```
Comment (boranx) [inline ci-operator/jobs/openshift/osd-network-verifier/openshift-osd-network-verifier-main-postsubmits.yaml:24]: just for knowledge: what does ci-operator do? How does it trigger coverage process?

Comment (bdematte) [reply ci-operator/jobs/openshift/osd-network-verifier/openshift-osd-network-verifier-main-postsubmits.yaml:24]: job files are generated by release jobs ( running `make update` before submitting the PR) so don't know precisely the internals :/ . I only updated myself the config file. 
But I need to update the secret names, I saw I didn't replace it properly before submitting :/

Comment (boranx) [reply ci-operator/jobs/openshift/osd-network-verifier/openshift-osd-network-verifier-main-postsubmits.yaml:24]: ack, thanks

**Review by boranx — COMMENTED** [review]
/lgtm

**Pull request merged**